release.yaml

```diff
@@ -16,8 +16,10 @@ fleet-crd:
 - 101.1.0+up0.6.0-rc.2
 neuvector:
 - 101.0.2+up2.4.0
+- 102.0.0+up2.4.2
```

Comment:
We should remove `101.0.2+up2.4.0` since we are increasing the major version to 102.x.y

Reply:
agree, we only release 1 app version per rancher version

Reply:
To remove a chart you need to run `CHART=<chart-name> VERSION=<chart-version> make remove`, remove the entry from release.yaml, and commit everything that was modified by the command. Then do the same steps to remove the crd chart in a separate commit.

Reply:
After make remove, do I need to do rebase as per this link. https://github.com/rancher/charts/blob/dev-v2.7/docs/developing.md#rebasing-an-existing-package

If so what version should I specify for the below line. I will contact you tomorrow via slack to take your help to fix this. 

3. Modify the package.yaml to point to your new upstream.

Reply:
@prachidamle @PennyScissors Removed the unreleased version 101.0.2+up2.4.0 and resubmitted the PR.

Reply:
@selvamt94 101.0.2+up2.4.0 should be removed from release.yaml file too